Java/src/main/java/com/gildedrose/BackstagePassItemUpdater.java

```diff
@@ -0,0 +1,57 @@
+package com.gildedrose;
+
+public class BackstagePassItemUpdater extends ItemUpdater {
+    @Override
+    void setQualityForValidItem(Item item) {
+        if (isSellInLessThanTenAndMoreThanFive(item) && isQualityLessThanFifty(item)) {
```

Comment:
Similar to the comment above - should `5` and `10` be hardcoded? 

edit: it applies to more instances across your PR :D